metashare/api/serializers.py

```diff
@@ -237,6 +280,22 @@ def get_pr_url(self, obj) -> Optional[str]:
             return f"https://github.com/{repo_owner}/{repo_name}/pull/{pr_number}"
         return None
 
+    def update(self, instance, validated_data):
+        user = getattr(self.context.get("request"), "user", None)
+        if user:
+            originating_user_id = str(user.id)
+        else:
+            originating_user_id = None
```

Comment:
@wlonk When testing this locally, I came back around to your initial idea of including this. There was a JSON serializing error with the initial implementation you had, so I added the `str()` wrapper.